spec/e2e/distance-matrix-spec.js

```diff
@@ -21,7 +21,8 @@ var objectContaining = jasmine.objectContaining;
 describe('distance matrix client library', function() {
   var googleMaps = require('./service');
 
-  var inOneHour = new Date().getTime() + 60 * 60 * 1000;
+  var now = new Date().getTime();
+  var inOneHour = Math.round(now/1000);
```

Comment:
is this still one hour ahead?

Reply:
oh sorry you are right I've corrected it